homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -59,8 +59,17 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up Fritz!Box call monitor sensor platform."""
     name = config.get(CONF_NAME)
+    host = config.get(CONF_HOST)
     # Try to resolve a hostname; if it is already an IP, it will be returned as-is
-    host = socket.gethostbyname(config.get(CONF_HOST))
+    try:
+        host = socket.gethostbyname(host)
+    except socket.error:
+        _LOGGER.warning(
+            "Could not resolve hostname: %s.\nTrying default address %s",
```

Comment:
Remove the part about the default host. If the lookup fails then the setup should fails. Assuming too much things make it hard to debug for users.

Reply:
done